README.md

```diff
@@ -12,33 +12,101 @@ They can also be used independently of Witan as a standalone demographic models
 
 Current status:
 * Population projections: First release of a minimal version coming soon!
-* Small area projections: Not started. Waiting to finish the population projections.
 
-See the [docs](doc/intro.md) for more info!
+See the [docs](https://github.com/MastodonC/witan.models.demography/blob/master/doc/intro.md) for more info about the methodology.
+
+## Requirements
+
+You need to have [Leiningen](http://leiningen.org/) installed to run the projections.
 
 ## Installation
 
-Download from http://example.com/FIXME.
+1) Download or clone this repository.
+
+2) Use the terminal to navigate to the directory that the repo is in, for example:
+```
+	$ cd witan.models.demography
+```
+
+3) Create an executable .jar file:
+```
+	$ lein clean
+
+	$ lein with-profile cli uberjar
+```
+
+Copy the filepath to the standalone file, starting at the `target` directory, to use later. For example:
```

Comment:
Yes, although from the previous step they should be in the project directory now. If they use the full filepath from inside the project directory does that cause a problem?